core/silkworm/common/util.cpp

```diff
@@ -83,20 +83,14 @@ evmc::bytes32 to_bytes32(ByteView bytes) {
     return out;
 }
 
-ByteView zeroless_view(const ByteView& data) {
+ByteView zeroless_view(ByteView data) {
```

Comment:
Why remove const reference ?
This way we have a new copy instance of data passed as argument and evenually a third instance returned as result

Reply:
Well, `ByteView` is just a pointer + size, it's cheap to pass it by value. Shouldn't matter much either way.

Reply:
Agree is lightweight but nevertheless there's a copy ... which is eventually havier than a ref.
Semantically then, as we also don't tamper with input data, I'd keep the const ref

Reply:
I also prefer to pass ByteView by copy, the method signature is more readable. The copy is very cheap: we have a pointer + length copy vs a pointer copy.

Reply:
Ok. I won't die on this hill